salt/modules/aptpkg.py

```diff
@@ -44,6 +44,7 @@
 import salt.utils.systemd
 import salt.utils.versions
 import salt.utils.yaml
+from salt.utils.environment import get_module_environment
```

Comment:
We either need to use the full module namespace, or we need to import the function as a private function (i.e. `as _get_module_environment`). The reason for this is because the loader will add all public functions in the module into the `__salt__` dunder.